Assets/New Babylon/Scripts/Monologues/SpeakerSound.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpeakerSound : AudioHandler
+{
+    //audio stuff
+    [Header("Speaker Bools")]
+    public bool hasVoiceAudio;
+    public bool usesAllLetters, countsUp;
+    public int speakFreq = 4;
+    
+    //for random sound reading
+    [Header("Speaker Sounds")]
+    public AudioClip[] spokenSounds;
+    //for spoken alphabet
+    public List<char> letters = new List<char>();
+    public List<char> capitalLetters = new List<char>();
+    //for matching up letters with sounds 
+    public List<AudioClip> spokenLetters = new List<AudioClip>();
+
+    //checks what kind of audio to play for the speaker 
+    public void AudioCheck(string lineOfText, int letter)
+    {
+        if (hasVoiceAudio)
+        {
+            if (!countsUp)
+            {
+                if (letter % speakFreq == 0)
+                    Speak(lineOfText[letter]);
```

Comment:
Consider flattening your if statements by using and or or conditions. 

If (hasVoiceAudio && !countsUp)